bam.iobio/src/App.tsx

```diff
@@ -0,0 +1,247 @@
+/*
+ *
+ * Copyright (c) 2024 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ *  This program and the accompanying materials are made available under the terms of
+ *  the GNU Affero General Public License v3.0. You should have received a copy of the
+ *  GNU Affero General Public License along with this program.
+ *   If not, see <http://www.gnu.org/licenses/>.
+ *
+ *  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ *  EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ *  OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ *  SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ *  INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ *  TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ *  OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ *  IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ *  ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+import { useState, useEffect } from 'react';
+import './App.css';
+
+// import bamClass from 'bam.iobio/client/js/class.js';
+
+// import d3 from 'bam.iobio/client/js/d3.min.js';
+// import bamd3 from 'bam.iobio/client/js/bam.d3.js';
+// import donut from 'bam.iobio/client/js/donut.d3.js';
+// import histogram from 'bam.iobio/client/js/histogram.d3.js';
+// import histogramViewFinder from 'bam.iobio/client/js/histogramViewFinder.d3.js';
+// import iobioViz from 'bam.iobio/client/js/iobio.viz.js';
+// import iobio from 'bam.iobio/client/js/iobio.min.js';
+// import nprogress from 'bam.iobio/client/js/nprogress.js';
+// import rdp from 'bam.iobio/client/js/rdp.js';
+// import movingLine from 'bam.iobio/client/js/movingLine.d3.js';
+// import socket from 'bam.iobio/client/js/socket.io.js';
+
+// import build from 'bam.iobio/client/dist/build.js';
```

Comment:
Putting this on hold until we sign off on basic repo setup. Extra work to integrate old scripts if Iobio is making a new library.